src/EventKit/EKEnums.cs

```diff
@@ -75,6 +75,16 @@ public enum EKErrorCode : long {
 		ProcedureAlarmsNotMutable,
 		EventStoreNotAuthorized,
 		OSNotSupported,
+		[iOS (13,0), Mac (10,15)]
+		InvalidInviteReplyCalendar,
+		[iOS (13,0), Mac (10,15)]
+		NotificationsCollectionFlagNotSet,
+		[iOS (13,0), Mac (10,15)]
+		SourceMismatch,
+		[iOS (13,0), Mac (10,15)]
+		NotificationCollectionMismatch,
+		[iOS (13,0), Mac (10,15)]
+		NotificationSavedWithoutCollection,
```

Comment:
I don't know which version of ```Watch```I should put.

Reply:
since this is added in Xcode 11 then it's the watchOS SDK version that was provided, i.e. 6.0